### DISCUSSION TITLE
Finding QR codes

### DISCUSSION THREAD

**Discussion by OmikeOo** [gopro/labs#905] [General]
I have a new GoPro12 and want to run it for long periods on a power bank. Have seen the info re low power warnings and the 'fix' using !MWAKE=2.Where can I find the QR code for this please. PS I'm pretty ignorant around this stuff!

**Comment by dnewman-gpsw** [comment]
*WAKE=2 makes the camera boot when power is added, nothing to do with low power.  Only low power want is related to bad or long USB cables, or many no power banks. For these situations use *TUSB=1, for trust USB.  These are documented here https://gopro.github.io/labs/control/extensions/

**Comment by OmikeOo** [comment]
Thanks for the reply, I appreciate  it. Thought I read, possibly on github
that TUSB didn't work with hero 12 and better to use !MWAKE? Using cable
supplied by Gopro with pass through door and good quality Anka power bank.
Thanks.

Mike

**Reply by dnewman-gpsw** [reply]
Many consumer power banks don't report their power rating it seems.  Nothing to do with WAKE

**Comment by OmikeOo** [comment]
Well I've found the QR Control app and set TUSB as 1 but have the same low
power message . After considering lots of other cameras and having had my
original GoPro for  over 10 years and it still works I'm beginning to
regret my choice here. There seems to be little if any support available
from Gopro. Have I wasted my money? I want to be able to record for a
minimum of 6 hours. My original go pro will still do over 4 hours on its
own battery pack. Have we gone backwards in the intervening years?

**Comment by OmikeOo** [comment]
Interesting video here. Is this the answer?

https://youtu.be/ehc8PNLzqes?si=5ye_D_wwUWpLZ6ps

**Reply by dnewman-gpsw** [reply]
Sure, like a said is it is bad cable or power bank not reporting its current rating. Also TUSB does work as I verified it just now.  There is no bug in Labs or HERO12 on this.  Only a few days about I did a 6+ hour timelapse for the Aurora. 

![image](https://github.com/gopro/labs/assets/10856084/5e8c3361-c7e7-4126-97c5-a1fe938854e4)
